arangod/RestHandler/RestSchemaHandler.cpp

```diff
@@ -0,0 +1,721 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2025-2025 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Business Source License 1.1 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     https://github.com/arangodb/arangodb/blob/devel/LICENSE
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Koichi Nakata
+////////////////////////////////////////////////////////////////////////////////
+
+#include "Aql/Query.h"
+#include "Aql/QueryRegistry.h"
+#include "Basics/voc-errors.h"
+#include "Futures/Utilities.h"
+#include "Graph/GraphManager.h"
+#include "IResearch/IResearchLink.h"
+#include "IResearch/IResearchLinkHelper.h"
+#include "IResearch/IResearchView.h"
+#include "Logger/LogMacros.h"
+#include "RestHandler/RestSchemaHandler.h"
+#include "RestHandler/RestVocbaseBaseHandler.h"
+#include "Transaction/Methods.h"
+#include "Transaction/OperationOrigin.h"
+#include "Transaction/StandaloneContext.h"
+#include "Utils/CollectionNameResolver.h"
+#include "VocBase/LogicalCollection.h"
+#include "VocBase/Methods/Indexes.h"
+
+#include <velocypack/Builder.h>
+#include <velocypack/vpack.h>
+
+using namespace arangodb;
+using namespace arangodb::rest;
+
+namespace {
+constexpr std::string_view moduleName("graph management");
```

Comment:
```suggestion
constexpr std::string_view moduleName("schema endpoint");
```

Reply:
The module name is to describe where the call came from.
So this should be the name of the Schema endpoint